packages/driver/test/cypress/integration/commands/querying_spec.coffee

```diff
@@ -1422,12 +1422,18 @@ describe "src/cy/commands/querying", ->
 
     describe "special characters", ->
       _.each "' \" [ ] { } . @ # $ % ^ & * ( ) , ; :".split(" "), (char) ->
-        it "finds content with character: #{char}", ->
+        it "finds content by string with character: #{char}", ->
           span = $("<span>special char #{char} content</span>").appendTo cy.$$("body")
 
           cy.contains("span", char).then ($span) ->
             expect($span.get(0)).to.eq span.get(0)
 
+        it "finds content by regex with character: #{char}", ->
+          span = $("<span>special char #{char} content</span>").appendTo cy.$$("body")
+
+          cy.contains("span", new RegExp(_.escapeRegExp(char))).then ($span) ->
```

Comment:
Awesome - thank you for taking the lead on this.

However, Isn't this obfuscating the actual bug? By escaping it in your argument here, this would pass even without your fix in `elements.coffee` right?

I believe you shouldn't be escaping the regexp here so that it proves we're correctly escaping it for you. I haven't run this though - so I could be totally off.

Reply:
it does not pass without the fix in elements.coffee (confirmed before submitting). `‘` is a valid non-special reflex character and is passed through excapeRegExp unchanged. the escapeRegExp is for the other characters